src/averageAges.js

```diff
@@ -36,8 +43,20 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
-function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+function calculateWomenAverageAge(people, withChildren = false) {
+  let women = people.filter(person =>
+    person.sex === 'f');
+
+  women = withChildren
```

Comment:
You can use a single filter with a conditional statement in the callback function to filter women based on the `withChildren` parameter.